### DISCUSSION TITLE
Installed the Ring-mqtt add on and configured but devices are not showing up

### DISCUSSION THREAD

**Discussion by sevanb** [tsightler/ring-mqtt#931] [Support Q&A]
Hello, I went through the installation of this add on for my ring cameras and sensors.  After logging in and authenticating none of the devices are showing up either in the entities or devices.  Any idea what I did wrong?  Do I need the regular ring add on from the store as well?  The log shows that the devices are being imported to Home Assistant.

[03cabcc9_ring_mqtt_2024-11-12T21-07-49.385Z.log](https://github.com/user-attachments/files/17722823/03cabcc9_ring_mqtt_2024-11-12T21-07-49.385Z.log)

**Comment by tsightler** [comment]
The logs show that ring-mqtt itself is doing what it is supposed to do, connecting to Ring and MQTT and sending messages between them MQTT.  That means the problem is on the other side of the MQTT broker, i.e. Home Assistant must also connect to the MQTT broker to be able to receive messages so my guess is that side is not working.  Make sure you have configured the MQTT integration and that it is working and, ideally, is using all defaults.

**Comment by sevanb** [comment]
Thanks for the response.  Here is the log from Mosquitto broker that I have configured.  

![MQTT broker](https://github.com/user-attachments/assets/b7b61d11-d417-4813-a0bb-c74309fff88a)

[core_mosquitto_2024-11-12T22-07-41.105Z.log](https://github.com/user-attachments/files/17723325/core_mosquitto_2024-11-12T22-07-41.105Z.log)
